Packs/GoogleWebRiskAPI/Integrations/GoogleWebRiskV1/GoogleWebRiskV1.py

```diff
@@ -0,0 +1,40 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+import requests
+from ast import literal_eval
+
+Apikey = demisto.params().get('APIkey')
+ThreatCategories = demisto.params().get('ThreatCategories')
+search = ''
+
+for item in ThreatCategories:
```

Comment:
Please move this block to a helper function.
call it from main.
If it fails throw an exception with a description.
invalid APIKey or invalid ThreatCategories.